Makefile

```diff
@@ -15,7 +15,7 @@ clean:
 requirements: clean-reqs requirements/requirements.txt requirements/requirements-test.txt requirements/requirements-dev.txt
 
 clean-reqs:
```

Comment:
I’m not sure if there is an actual use-case for this target. (Not something about the actual change made in this PR though.)

Reply:
yes, probably not.

I just added `-f` because in case you do not have the files yet and want to run `make requirements` the `clean-reqs` will fail.

but the files get overwritten by the `requirements/*` tasks anyways...